src/calculateRentalCost.js

```diff
@@ -29,6 +29,19 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let cost = 0;
+
+  for (let i = 0; i <= days; i++) {
+    if (days < 3) {
+      cost = 40 * days;
```

Comment:
1. You calculate `40 * days` three times. Do it once and use variable.
2. You change one variable and return it right away. Get rid of it and put `return` statement right inside `if` structure. That way you can also avoid using `else`.